mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -342,19 +423,25 @@ def select_color(self):
                     self.error_dialog = QtWidgets.QErrorMessage()
                     self.error_dialog.showMessage('Use "options" to change color of an activated flighttrack.')
                 else:
-                    color = QtWidgets.QColorDialog.getColor()
-                    if color.isValid():
-                        self.dict_flighttrack[wp_model]["color"] = color.getRgbF()
-                        self.color_change = True
-                        self.list_flighttrack.currentItem().setIcon(self.show_color_icon(self.get_color(wp_model)))
-                        self.dict_flighttrack[wp_model]["patch"].update(color=self.dict_flighttrack[wp_model]["color"])
+                    color_dialog = CustomColorDialog(self)
+                    color_dialog.color_selected.connect(lambda color: self.apply_color(wp_model, color))
+                    color_dialog.exec_()
             else:
-                self.labelStatus.setText("Check Mark the flighttrack to change its color.")
+                self.labelStatus.setText("Status: Check Mark the flight track to change it's Colour.")
```

Comment:
Not sure about "flighttrack" vs "flight track", but "its" without an apostrophe is correct. "color" vs "colour" is just american-influenced vs british-influenced english, and in any case should be lowercase.